### PR TITLE
Add tests to deposit managements

### PR DESCRIPTION
**Pull request by JozefImiolo**
Please review my code and implements methods

### CODE REVIEW

File: src/main/java/progob/Account.java [hunk 1]
```diff
@@ -1,9 +1,13 @@
 package progob;
 
+import java.util.*;
```
Comment (todrzywolek) [inline src/main/java/progob/Account.java]: Why do we need this import? It seems reduntant to me. Please remove it.

File: src/test/java/progob/BankTest.java [hunk 2]
```diff
@@ -95,4 +95,17 @@ public void testGetClientAccounts()
         assertTrue(bank.getClientAccounts(klient1).contains(konto_Klienta2));
         assertFalse(bank.getClientAccounts(klient1).contains(konto_Klienta3));
     }
+    public void testDeposits() throws WithdrawException
```
Comment (todrzywolek) [inline src/test/java/progob/BankTest.java]: Test method name should inform what we are testing. Please change test method name for something more meaningful for example: "testGetClientDeposits_When_ClientHasTwoDepositsInBank"

File: src/test/java/progob/DepositTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    private Account konto;
+    private Deposit deposit;
+    @Before
+    public void setup() {
+        Client klient = new Client("Kamil", "Nowak");
+        konto = new Account(klient);
+    }
+
+    @Test
+    public void testDeposits() throws WithdrawException {
```
Comment (todrzywolek) [inline src/test/java/progob/DepositTest.java]: Test method name should inform what we are testing. Please change test method name for something more meaningful

**Review by todrzywolek — CHANGES_REQUESTED** [review]
I have reviewed the code. Please apply these changes before merging

File: src/main/java/progob/Account.java (hunk 1 above)
Comment (JozefImiolo) [reply src/main/java/progob/Account.java]: I removed it

File: src/test/java/progob/BankTest.java (hunk 2 above)
Comment (JozefImiolo) [reply src/test/java/progob/BankTest.java]: I change name of this method

File: src/test/java/progob/DepositTest.java (hunk 3 above)
Comment (JozefImiolo) [reply src/test/java/progob/DepositTest.java]: I changed test method name on testCreateDeposit

**Pull request merged**